src/generators/default/class-expr-static-private-method.template

```diff
@@ -0,0 +1,48 @@
+// Copyright (C) 2018 Jaideep Bhoosreddy. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+/*---
+path: language/expressions/class/gen-private-method-static-
+name: Static generator private method as a ClassExpression element
+features: [class-methods-private]
+esid: prod-GeneratorPrivateMethod
+info: |
+  ClassElement :
+    static PrivateMethodDefinition
+
+  MethodDefinition :
+    GeneratorMethod
+
+  14.4 Generator Function Definitions
+
+  GeneratorMethod :
+    * PropertyName ( UniqueFormalParameters ) { GeneratorBody }
+features: [generators]
+---*/
+
+var callCount = 0;
+
+var C = class {
+    static *gen() {
+        callCount += 1;
+        /*{ body }*/
+    }
+    static gen() {
+        return C.gen();
+    }
+}
+
+var gen = C.gen;
```

Comment:
It should be more adequate if you just use `var iter = C.gen()` (on line 40).

Something else that might simplify is an accessor method:

```js
var C = class {
  static *#gen() {
    /* ... */
  }

  static get gen() { return this.#gen; }
}

var iter = C.gen();
```